app/controllers/meetings/publish/notulen.js

```diff
@@ -99,29 +100,45 @@ export default class MeetingsPublishNotulenController extends Controller {
     return !!this.publishedResource;
   }
 
+  async loadSignedResources(versionedNotulenId) {
+    const signedResources = await this.store.query('signed-resource', {
```

Comment:
I would do the filter in the query

It's almost impossible to go wrong _in this specific scenario_ cause you can't really create more than 2 deleted signatures in sequence

but if there _would_ be `>pagesize` (aka more than 20) deleted signed resources that were newer than the last undeleted one, you would get no undeleted resources in your result.

...which is pretty much impossible here unless something goes quite badly wrong. But still, it's good practice to filter on the backend